lab/polymer/index.html

```diff
@@ -5,33 +5,31 @@
   <meta charset="utf-8">
   <title>Palindrom with Polymer demo</title>
   <!-- include web components polyfill -->
-  <script src="../../../webcomponentsjs/webcomponents.js"></script>
+  <script src="../utils/webcomponents.js"></script>
 
 
   <!-- include Palindrom with dependencies -->
-  <script src="../../../fast-json-patch/src/json-patch-duplex.js"></script>
-  <script src="../../src/palindrom.js"></script>
-  <script src="../../src/palindrom-dom.js"></script>
+  <script src="../../dist/palindrom-dom.js"></script>
 
   <!-- include components for demo -->
-  <link rel="import" href="../../../polymer/polymer.html">
+  <link rel="import" href="polymer/polymer.html">
 
   <!-- include polyfills -->
   <script src="polyfills/object-observe.js"></script>
   <script src="polyfills/array-observe.js"></script>
-  <link rel="import" href="../../../dom-bind-notifier/dom-bind-notifier.html">
+  <link rel="import" href="dom-bind-notifier/dom-bind-notifier.html">
```

Comment:
@alshakero I am quite surprised to see `dom-bind-notifier`, that was a bridge element that was used for Polymer 1.0-1.1, but later was deprecated by us. It should not take any part in the current demo.